### PR TITLE
enh(logs): Managing the modification of the log_id column from the centreon_storage.logs table

### PR DESCRIPTION
**Pull request by callapa**
## Description

The purpose of this procedure is to allow modification of log_id column of centreon_storage.logs table.

**Fixes** # (issue)

Fix the limitation of max value for the primary key of the centreon_storage.logs table

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [x] Updating documentation (missing information, typo...)

## Target serie

- [ ] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [x] 19.10.x (master)

<h2> How this pull request can be tested ? </h2>

This script will:

* Stop broker
* Rename the old log table (logs => logs_old)
* Create a new log table without the log_id column (=> logs)
* Restart Broker
* Copy the data from the old log table to the new one

**Pay attention, this script requires the root password of the database**

#### Community contributors & Centreon team

- [x] I followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: tools/update_centreon_storage_logs.php [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+    }
+
+    if (! isset($dbPassword)) {
+        // We need a root access
+        $dbPassword = askQuestion("Please enter the root password of database: ", true);
+    }
+
+    try {
+        $dsn = sprintf(
+            "mysql:dbname=centreon_storage;host=%s;port=%d",
```
Comment (kduret) [inline tools/update_centreon_storage_logs.php]: db name shoud be get from config file

File: doc/en/faq/known_issues/update_centreon_storage_logs.rst [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    root password of Centreon database (eg. --password=my_root_password).
+  --keep:
+    keep data of old table to centreon_storage.logs_old.
+  --no-keep:
+    remove progressively data from centreon_storage.logs_old during the migration to the new table centreon_storage.logs.
+  --temporary-path:
+    directory in which temporary files will be stored
+
+.. warning::
+  If you decide to keep the data from the old centreon_storage.logs table, do not forget to check the available disk space.
```
Comment (thiuyendang) [inline doc/en/faq/known_issues/update_centreon_storage_logs.rst]: Il manque le paragraphe sur la reprise du traitement si jamais il a été interrompu

File: doc/fr/faq/known_issues/update_centreon_storage_logs.rst [hunk 3]
```diff
@@ -0,0 +1,100 @@
+.. _update_centreon_storage_logs:
+
+=============================================
+Mise à jour de la table centreon_storage.logs
+=============================================
+
+Problématique
+===========
+
+Certains clients ont atteint le nombre maximal d'enregistrements dans la table centreon_storage.logs qui actuellement ne peux contenir que 2 147 483 647 enregistrements (entier signé, comptage à partir de 0).
```
Comment (thiuyendang) [inline doc/fr/faq/known_issues/update_centreon_storage_logs.rst]: ne peut

File: doc/fr/faq/known_issues/update_centreon_storage_logs.rst [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    mot de passe root de la base de données Centreon (ex. --password=my_root_password).
+  --keep:
+    indique que faut conserver les données de l’ancienne table (renommé en centreon_storage.logs_old).
+  --no-keep:
+    indique que les données de l’ancienne table centreon_storage.logs_old peuvent être supprimées au fur et à mesure de la migration des données vers la nouvelle table centreon_storage.logs.
+  --temporary-path:
+    indique le dossier où seront stockés les fichiers temporaires.
+
+.. warning::
+  Si vous décidez de conserver les données de l'ancienne table centreon_storage.logs n'oubliez pas de vérifier l'espace disque disponible.
```
Comment (thiuyendang) [inline doc/fr/faq/known_issues/update_centreon_storage_logs.rst]: paragraphe manquant par rapport à la doc Confluence pour les options de reprise

File: doc/en/faq/known_issues/update_centreon_storage_logs.rst [hunk 5]
```diff
@@ -0,0 +1,100 @@
+.. _update_centreon_storage_logs:
+
+==================================
+Update centreon_storage.logs table
+==================================
+
+Issues
+======
+
+Some customers have reached the maximum number of records in the centeron_storage.logs table which currently can only contain 2 147 483 647 records (signed integer, count from 0).
```
Comment (thiuyendang) [inline doc/en/faq/known_issues/update_centreon_storage_logs.rst]: centreon_storage.logs

File: doc/en/faq/known_issues/update_centreon_storage_logs.rst [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+Broker can therefore no longer add any elements to this table.
+
+Objectif
+========
+
+The purpose of this procedure is to allow the modification of the log_id column of the centeron_storage.logs table.
```
Comment (thiuyendang) [inline doc/en/faq/known_issues/update_centreon_storage_logs.rst]: centreon_storage.logs table.

**Pull request merged**